### PR TITLE
[somfytahoma] bug fixes

### PR DESCRIPTION
**Pull request by octa22**
This is a bugfix version:
- fixed leakage of a scheduler during discovery
- fixed 2+ concurrent bridges malfunction because of one HttpClient sharing 
- fixed non working action groups

### CODE REVIEW

File: bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java [hunk 1]
```diff
@@ -115,6 +116,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     public void initialize() {
         thingConfig = getConfigAs(SomfyTahomaConfig.class);
 
+        try {
+            httpClient.start();
+        } catch (Exception e) {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java]: Can you please specify the exceptions you expect to catch here.

File: bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java [hunk 2]
```diff
@@ -307,6 +315,13 @@ private void cleanup() {
         logger.debug("Doing cleanup");
         stopPolling();
         executions.clear();
+        if (httpClient != null) {
+            try {
+                httpClient.stop();
+            } catch (Exception e) {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java]: see above

File: bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java (hunk 1 above)
Comment (octa22) [reply bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java]: Sorry, the exception is generic one...

protected void start​(LifeCycle l) throws java.lang.Exception
Starts the given lifecycle.
Parameters:
l - the lifecycle to start
Throws:
java.lang.Exception - if unable to start lifecycle

File: bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java (hunk 2 above)
Comment (octa22) [reply bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java]: the same as above, it just throws a generic exception

**Pull request merged**